src/openrct2/ride/RideTypes.h

```diff
@@ -10,9 +10,10 @@
 #pragma once
 
 #include <cstdint>
+#include <limits>
 
-typedef uint8_t ride_id_t;
-typedef uint16_t ride_idnew_t; // Temporary, old one can be removed after switching save format.
+typedef uint16_t ride_id_t;
```

Comment:
Now's as good a time as any to switch to C++ style `using` instead of C style `typedef`.

Reply:
```suggestion
using ride_id_t = uint16_t;
```